integreat_cms/api/v3/locations.py

```diff
@@ -70,6 +70,11 @@ def transform_poi_translation(poi_translation: POITranslation) -> dict[str, Any]
     """
 
     poi = poi_translation.poi
+
+    primary_contact = Contact.objects.filter(
+        location=poi, point_of_contact_for="", name=""
```

Comment:
```suggestion
        location=poi, point_of_contact_for=""
```

Sorry, that was again a mistake in the issue description. The definition of "primary contact" is contact witohut title (point_of _contact_for) 🙈